Plugin.cs

```diff
@@ -133,6 +137,36 @@ private void OnCommand(string command, string args)
                     this.configuration.Save();
                     PrintSetting($"{Name} {verboseName}", this.configuration.verbose);
                     return;
+                case whitelistName:
+                    if(argArray.Length == 1){
+                        string whitelistString = string.Join(", ", this.configuration.whitelist);
+
+                        this.pi.Framework.Gui.Chat.Print($"Retainer whitelist: {whitelistString}");
+                        return;
+                    }
```

Comment:
If you're going to add an additional command to show the list, mention it in the README and the help dialog.